datafusion/core/tests/sql/information_schema.rs

```diff
@@ -30,91 +30,6 @@ use rstest::rstest;
 
 use super::*;
 
-#[tokio::test]
```

Comment:
I ported a few tests over to show the behavior -- I will port what else I can from this file as a follow on PR